resources/responses.js

```diff
@@ -0,0 +1,523 @@
+      fr: 'Gauche',
+      cn: '左',
+    };
+    return obj;
+  },
+  goWest: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: '<= Get Left/West',
+      fr: '<= Allez à Gauche/Ouest',
+      de: '<= Nach Links/Westen',
+      ko: '<= 왼쪽이 안전',
+      ja: '<= 左/西へ',
+    };
+    return obj;
+  },
+  goEast: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Get Right/East =>',
+      fr: 'Allez à Droite/Est =>',
+      de: 'Nach Rechts/Westen =>',
+      ko: '오른쪽이 안전 =>',
+      ja: '右/東へ =>',
+    };
+    return obj;
+  },
+  goFrontBack: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Go Front/Back',
+      de: 'Geh nach Vorne/ Hinten',
+      fr: 'Devant/Derrière',
+    };
+    return obj;
+  },
+  goSides: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Sides',
+      de: 'Seiten',
+      fr: 'Côtés',
+      ja: '横へ',
+    };
+    return obj;
+  },
+  outOfFront: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Out of Front',
+      de: 'Weg von Vorne',
+      fr: 'Ne restez pas devant',
+    };
+    return obj;
+  },
+  killExtraAdd: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Kill Extra Add',
+      de: 'Add angreifen',
+      ja: '水の精倒して',
+      fr: 'Tuez l\'add',
+      cn: '击杀小怪',
+    };
+    return obj;
+  },
+  killAdds: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Kill adds',
+    };
+    return obj;
+  },
+  awayFromFront: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Away From Front',
+      de: 'Weg von Vorne',
+      fr: 'Eloignez vous de l\'avant',
+    };
+    return obj;
+  },
+  silence: (sev) => {
+    let obj = {};
+    obj[defaultInfoText(sev)] = {
+      en: 'Silence',
+      de: 'Verstummen',
+      fr: 'Silence',
+      ja: 'ストンスキン',
+      cn: '沉默石肤',
+    };
+    return obj;
+  },
+  preyOn: (sev) => {
+    (data, matches) => {
+      let target = getTarget(matches);
+      if (data.me == target) {
+        return {
+          en: 'Prey on YOU',
+          de: 'Rakete auf DIR',
+          fr: 'Marquage sur VOUS',
+          ko: '빨간징 → 나',
+          ja: 'マーカー on YOU',
+        };
+      }
+      return {
+        en: 'Prey on ' + data.ShortName(target),
+        de: 'Beute auf ' + data.ShortName(target),
+        fr: 'Marquage sur ' + data.ShortName(target),
+        ko: '빨간징 → ' + data.ShortName(target),
+        ja: 'マーカー on ' + data.ShortName(target),
+      };
+    };
+    return obj;
+  },
+  meteorOnYou: (sev) => {
+    let obj = {};
+    obj[defaultAlarmText(sev)] = {
+      en: 'Meteor on YOU',
+      de: 'Meteor auf DIR',
+      fr: 'Météore sur VOUS',
+      cn: '陨石点名',
+    };
+    return obj;
+  },
+  stop: (sev) => {
+    let obj = {};
+    obj[defaultAlertText(sev)] = {
+      en: 'Stop',
+      de: 'Stopp',
+      fr: 'Stop',
+    };
+    return obj;
+  },
```

Comment:
Would it be worth splitting this into "stop everything" vs "stop moving"? I'm thinking here of Acceleration Bomb as a "stop everything" mechanic, while there are various other mechanics (such as Scathach's Shadow Links or Guardian's Electric Pulse/Radar) that permit actions while forbidding movement.

Reply:
Yeah, good thought, thanks! I'll add both of those.